README.md

```diff
@@ -269,6 +269,21 @@ export default class App {
 }
 ```
 
+#### Decorators
```

Comment:
This is the Decorators sub-section for the Providers topic. If you look at the doc as a whole it makes sense.